spec.bs

```diff
@@ -5309,13 +5352,17 @@ scored bids. It's a [=struct=] with the following [=struct/items=]:
 <dl dfn-for="leading bid info">
   : <dfn>top score</dfn>
   :: A {{double}}, initially 0.0. The highest score so far.
+  : <dfn>top non-k-anon score</dfn>
+  :: A {{double}}, initially 0.0. The highest score so far when disregarding k-anonymity.
   : <dfn>top bids count</dfn>
   :: An integer, initially 0. The number of bids with the same `top score`.
   : <dfn>at most one top bid owner</dfn>
   :: A [=boolean=], initially true. Whether all bids of `top score` are from the same interest
     group owner.
   : <dfn>leading bid</dfn>
-  :: Null or a [=generate bid=], initially null. The leading bid of the auction so far.
+  :: Null or a [=generated bid=], initially null. The leading bid of the auction so far.
+  : <dfn>leading non-k-anon bid</dfn>

```

Comment:
want to clarify on this: leading bid must be k-anon, and this `leading non-k-anon bid` just disregards k-anon and can be either k-anon or non-k-anon. Is that correct?

Reply:
Yes. It's a frequent problem, "leading bid disregarding k-anon" is a bit of a mouthful...

Reply:
... I probably should change it to something like that though?

Reply:
And done